src/audio/selector/selector_generic.c

```diff
@@ -23,12 +23,13 @@ LOG_MODULE_DECLARE(selector, CONFIG_SOF_LOG_LEVEL);
 #define BYTES_TO_S16_SAMPLES	1
 #define BYTES_TO_S32_SAMPLES	2
 
+#if CONFIG_IPC_MAJOR_3
 #if CONFIG_FORMAT_S16LE
 /**
  * \brief Channel selection for 16 bit, 1 channel data format.
  * \param[in,out] dev Selector base component device.
  * \param[in,out] sink Destination buffer.
- * \param[in,out] source Source buffer.
+ * \param[in,out] source Src buffer.
```

Comment:
Why are you changing this? How is "src" better than "source" in a comment, especially since "destination" is used as a full word instead of "dst" above?

Reply:
CI reports warning that duplicated word